frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,216 @@
+              </v-row>
+            </div>
+          </v-card>
+        </v-card>
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col class="">
+              <AppLabel>Does your facility's not-for-profit have an open membership?</AppLabel>
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col>
+              <v-radio-group v-model="model.openMembership" :readonly="false" :rules="[...rules.required]" hide-details>
+                <v-row no-gutters>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="Yes" :value="1"></v-radio>
+                  </v-col>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="No" :value="0"></v-radio>
+                  </v-col>
+                </v-row>
+              </v-radio-group>
+            </v-col>
+          </v-row>
+        </v-card>
+
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col class="">
+              <AppLabel>Does your facility's not-for-profit have elected, unpaid board members?</AppLabel>
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col>
+              <v-radio-group v-model="model.boardMembersElected" :readonly="false" :rules="[...rules.required]" hide-details>
+                <v-row no-gutters>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="Yes" :value="1"></v-radio>
+                  </v-col>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="No" :value="0"></v-radio>
+                  </v-col>
+                </v-row>
+              </v-radio-group>
+            </v-col>
+          </v-row>
+        </v-card>
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col class="">
+              <AppLabel>Does your facility's not-for-profit have board members selected from the entire membership?</AppLabel>
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col>
+              <v-radio-group v-model="model.boardMembersSelectedMembership" :readonly="false" :rules="[...rules.required]" hide-details>
+                <v-row no-gutters>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="Yes" :value="1"></v-radio>
+                  </v-col>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="No" :value="0"></v-radio>
+                  </v-col>
+                </v-row>
+              </v-radio-group>
+            </v-col>
+          </v-row>
+        </v-card>
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
```

Comment:
do we need v-row and v-col here? probably just AppLabel and v-radio-group should be enough

Reply:
done